external/sql/storm-sql-core/src/codegen/includes/parserImpls.ftl

```diff
@@ -83,4 +83,23 @@ SqlNode SqlCreateTable() :
         input_format_class_name, output_format_class_name, location,
         tbl_properties, select);
     }
+}
+
+/**
+ * CREATE FUNCTION functionname AS 'classname'
+ */
+SqlNode SqlCreateFunction() :
+{
+    SqlParserPos pos;
+    SqlIdentifier functionName;
+    SqlNode className;
+}
+{
+    <CREATE> { pos = getPos(); }
+    <FUNCTION>
+    functionName = CompoundIdentifier()
+    <AS>
+    className = StringLiteral() {
```

Comment:
CREATE FUNCTION functionname AS 'classname' does not take any jar location or name. Where should user put UDF libs? Is it mandatory to put each UDF class in a separate jar? 
Can you please describe how should UDF libraries packaged and placed in storm environment?